lib/LoginStrategy.ts

```diff
@@ -239,4 +255,50 @@ export class LoginStrategy extends BaseStrategy {
       return acc
     }, [] as Claim[])
   }
-}
+
+  /**
+   * @description checks if the claimer is authenticated into the DID Document
+   *
+   * @param token
+   * @param payload
+   *
+   * @returns {string} issuer DID or null
+   */
+  async verifyClaim(token: string, payload: ITokenPayload): Promise<string> {
+    const didDocument = await this.cacheServerClient?.getDidDocument(payload.iss) ?? await this.didResolver.read(payload.iss)
+
+    if (this.isAuthorized(token, didDocument))
+      return payload.iss
+    return null 
+  }
+  
+  private isAuthorized = (claimedToken: string,didDocument: IDIDDocument) : boolean => {
+    //read publickey field in DID document
+    const didPubKey = didDocument.publicKey
+    if (didPubKey.length === 0)
+      return false
+    
+    const verifyEthersSignature = verificationMethods[1];
+
+    //get all authenticated public keys
+    const authenticatedPubkeys = didDocument.publicKey.filter(pubkey => {
+      return this.isAuthenticated(pubkey, didDocument.authentication)
+    })
+
+    //Keep in an array all keys passing the verifying process
+    const arekeysValid = authenticatedPubkeys.filter(async (key) => {
+      const publicKey = key["publicKeyHex"]
+      const keyStatus = await verifyEthersSignature(claimedToken, publicKey)
```

Comment:
Why using `verifyEtherSignature` directly instead of `JWT` class? (Not saying it's wrong necessarily, just trying to understand 😃 )

Reply:
If possible, I think it is slightly better to use the `JWT` class as that is the one that is exported cleanly from the `ew-did-registry/jwt` package. Not a big deal though.